### PR TITLE
Add data export/import section to en_US/operations/backup-restore.md

### PR DESCRIPTION
**Pull request by SergeTupchiy**

### CODE REVIEW

File: zh_CN/operations/backup-restore.md [hunk 1]
```diff
@@ -42,3 +42,92 @@ EMQX 默认使用 Mnesia 数据库来存储数据，可以直接备份 Mnesia 
...
+* 内置数据库 (Mnesia) 的数据
+  * 仪表盘用户和 API 密钥
+  * 用户身份凭证信息认证 （简单和增强）
+  * PSK 认证数据
+  * 授权规则
+  * 禁用客户端
+* 存储在 EMQX 数据目录（`node.data_dir`）中的 SSL/TLS 证书
+* 存储在 EMQX 数据目录中的授权 acl.conf 文件
+
+::: 警告
```
Comment (wivwiv) [inline zh_CN/operations/backup-restore.md]: Do not warn users
```suggestion
::: 特别提示
```

File: zh_CN/operations/backup-restore.md [hunk 2]
```diff
@@ -42,3 +42,92 @@ EMQX 默认使用 Mnesia 数据库来存储数据，可以直接备份 Mnesia 
...
+
+### 导出
+
+数据可以从任何运行的集群节点导出。
+
+### 导入
+
+要导入数据，EMQX 节点必须处于运行状态，并且需要满足一些条件才能成功进行导入操作：
+
+* 如果启用了[核心节点 + 复制者节点](../deploy/cluster/mria-introduction.md)模式，数据导入只能在核心节点上启动。这不会影响实际的导入行为，因为数据将被复制到所有集群节点，包括核心节点和复制者节点。在核心节点上启动导入操作可以确保正确协调操作。
```
Comment (wivwiv) [inline zh_CN/operations/backup-restore.md]: 复制者 -> 复制
```suggestion
* 如果启用了[核心节点 + 复制节点](../deploy/cluster/mria-introduction.md)模式，数据导入只能在核心节点上进行。这不会影响实际的导入行为，因为数据将被复制到所有集群节点，包括核心节点和复制节点。在核心节点上进行操作可以确保正确导入数据。
```

File: zh_CN/operations/backup-restore.md [hunk 3]
```diff
@@ -42,3 +42,92 @@ EMQX 默认使用 Mnesia 数据库来存储数据，可以直接备份 Mnesia 
...
+* 从 EMQX 企业版集群导出的数据无法导入到 EMQX 开源版集群。
+* 数据文件不能被重命名。
+
+如果上述任何条件不满足，导入过程将被中止，并显示适当的错误消息。
```
Comment (wivwiv) [inline zh_CN/operations/backup-restore.md]:
```suggestion
如果上述任何条件不满足，导入过程将被中止，并显示对应的错误消息。
```

File: zh_CN/operations/backup-restore.md [hunk 4]
```diff
@@ -42,3 +42,92 @@ EMQX 默认使用 Mnesia 数据库来存储数据，可以直接备份 Mnesia 
...
+
+在数据导入操作期间，数据将被插入（如果在目标 EMQX 集群中不存在）或更新（如果存在冲突）。导入过程不会从 EMQX 集群中删除任何现有数据。
```
Comment (wivwiv) [inline zh_CN/operations/backup-restore.md]:
```suggestion
在数据导入操作期间，数据将被插入（如果在目标 EMQX 集群中不存在）或更新（如果存在冲突）到 EMQX 中。导入过程不会从 EMQX 集群中删除任何现有数据。
```

File: zh_CN/operations/backup-restore.md [hunk 5]
```diff
@@ -42,3 +42,92 @@ EMQX 默认使用 Mnesia 数据库来存储数据，可以直接备份 Mnesia 
...
+
+::: tip 警告
```
Comment (wivwiv) [inline zh_CN/operations/backup-restore.md]:
```suggestion
::: tip 特别提示
```

File: en_US/operations/backup-restore.md [hunk 6]
```diff
@@ -37,3 +37,89 @@ Before EMQX 5.0, persistent sessions were stored in memory, and for the EMQX ent
...
+    Exporting emqx_admin database table...
+    Exporting emqx_authn_mnesia database table...
+    Exporting emqx_enhanced_authn_scram_mnesia database table...
+    Exporting emqx_app database table...
+    Exporting emqx_acl database table...
+    Exporting emqx_psk database table...
+    Exporting emqx_banned database table...
+    Data has been successfully exported to data/backup/emqx-export-2023-06-19-15-14-19.947.tar.gz.
+    ```
+2. Import data. The name of the imported file should be specified as an absolute path, e.g.:
```
Comment (wivwiv) [inline en_US/operations/backup-restore.md]: @SergeTupchiy I test that relative paths are also work, if you can give priority to the user to use relative paths (this may be simpler).

File: en_US/operations/backup-restore.md [hunk 7]
```diff
@@ -37,3 +37,89 @@ Before EMQX 5.0, persistent sessions were stored in memory, and for the EMQX ent
 ## Data Restore
 
 To restore the configuration file, you only need to place the backed-up configuration files and data in the corresponding directory before starting EMQX.
+
+## Data Import and Export
+
+EMQX 5.1 introduces a user-friendly tool for data import and export. While similar to the one available in EMQX 4.x, it has several significant differences and is not compatible with it.
```
Comment (Meggielqk) [inline en_US/operations/backup-restore.md]:
```suggestion
EMQX 5.1 introduces a user-friendly command-line tool for data import and export. While similar to the one available in EMQX 4.x, it has several significant differences and is not compatible with it.
```

File: zh_CN/operations/backup-restore.md [hunk 8]
```diff
@@ -42,3 +42,92 @@ EMQX 默认使用 Mnesia 数据库来存储数据，可以直接备份 Mnesia 
 ## 数据恢复
 
 在 EMQX 启动之前，将配置文件与数据放置到对应目录即可。
+
+## 数据导入和导出
+
+EMQX 5.1 引入了一个用户友好的数据导入和导出工具。尽管与 EMQX 4.x 中提供的工具类似，但它具有一些显著的差异，并且与之不兼容。
```
Comment (Meggielqk) [inline zh_CN/operations/backup-restore.md]:
```suggestion
EMQX 5.1 引入了一个用户友好的命令行工具用于数据导入和导出。尽管与 EMQX 4.x 中提供的工具类似，但它具有一些显著的差异，并且与之不兼容。
```

File: zh_CN/operations/backup-restore.md [hunk 9]
```diff
@@ -42,3 +42,92 @@ EMQX 默认使用 Mnesia 数据库来存储数据，可以直接备份 Mnesia 
...
+* 从 EMQX 企业版集群导出的数据无法导入到 EMQX 开源版集群。
+* 数据文件不能被重命名。
+
+如果上述任何条件不满足，导入过程将被中止，并显示适当的错误消息。
+
+在数据导入操作期间，数据将被插入（如果在目标 EMQX 集群中不存在）或更新（如果存在冲突）。导入过程不会从 EMQX 集群中删除任何现有数据。
+
+::: tip 警告
+
+在极少数情况下，现有数据可能与导入的数据不兼容。例如，如果 EMQX 集群使用内置数据库的身份认证机制，并具有“后缀”盐位置，而导入的数据定义了具有“前缀”盐位置的相同身份认证源，那么在导入之前创建的旧用户身份凭证将不再起作用。因此，将数据导入到未清除数据的 EMQX 集群可能需要额外的注意。
```
Comment (wivwiv) [inline zh_CN/operations/backup-restore.md]:
```suggestion
在极少数情况下，现有数据可能与导入的数据不兼容。
例如，某个 EMQX 集群使用了内置数据库认证，并设置盐的位置为“后缀”，而导入的数据将相同配置设置为”前缀“。导入之后新的配置将生效，之前创建的旧用户身份凭证将不再起作用。

因此，将数据导入到未清除数据的 EMQX 集群可能需要额外的注意。
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: zh_CN/operations/backup-restore.md [hunk 10]
```diff
@@ -1,44 +1,96 @@
...
 尽管如此，我们仍然应当做最坏的打算来确保数据万无一失，本章节指导您如何进行 EMQX 数据备份和恢复。
 
-## 配置文件
+## 数据导入导出介绍
 
-当使用配置管理工具、配置中心或 Git 管理配置时，配置的备份恢复通常不需要手动操作。
+EMQX 5.1 引入了一个用户友好的命令行工具用于数据导入和导出。尽管与 EMQX 4.x 中提供的工具类似，但它具有一些显著的差异，并且与之不兼容。
 
-但如果手动修改配置文件则需要将其定期备份，由于安装方式不同，配置文件所在位置也不同：
+在 EMQX 4.x 中，使用单个 JSON 文件携带了 EMQX 配置和内置数据库的所有必要数据。然而，在 EMQX 5.1 中，使用了一个不透明的压缩 tar 存档，这样可以更高效、更有结构地处理潜在的大量用户数据。
```
Comment (wivwiv) [inline zh_CN/operations/backup-restore.md]:
```suggestion
在 EMQX 4.x 中，使用单个 JSON 文件保存 EMQX 配置和内置数据库的所有必要数据。然而，在 EMQX 5.1 中，将导出的数据压缩为 tar 格式的文件，这样可以更高效、更有结构地处理潜在的大量用户数据。
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: en_US/operations/backup-restore.md (hunk 6 above)
Comment (SergeTupchiy) [reply en_US/operations/backup-restore.md]: Thanks @wivwiv , we indeed explicitly support relative paths and even a file basename, if it resides in `<EMQX data dir>/backup`.
I've updated it and also added two more examples, hopefully it won't be too wordy..

**Review by wivwiv — APPROVED** [review]

**Comment by SergeTupchiy** [comment]
@wivwiv , @Meggielqk thanks a lot for improving the text and review! :raised_hands:
